chapters/annotations.tex

```diff
@@ -523,15 +545,19 @@ \section{Simulations}\label{annotations-for-simulations}\label{simulations}
 \end{annotationdefinition}
 
 \begin{annotationdefinition}[TestCase]
-\begin{synopsis}[grammar]\begin{lstlisting}
-"TestCase" "(" "shouldPass" "=" ( false | true ) ")"
+\begin{synopsis}\begin{lstlisting}
+record TestCase
+  /*literal*/ constant Boolean shouldPass;
+end TestCase;
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
 If \lstinline!shouldPass! is \lstinline!false! it indicates that the translation or the simulation of the model should fail.
 If a tools checks a package where classes have \lstinline!shouldPass = false! they should not generate errors, and checking may even be skipped.
 On the other hand, models with \lstinline!shouldPass = false! may be useful for creation of negative tests in tool-specific ways.
 Similarly as a class with \lstinline!obsolete! annotation, a class with \lstinline!TestCase! annotation (regardless of the value of \lstinline!shouldPass!) shall not be used in other models, unless those models also have a \lstinline!TestCase! annotation.
 
+If the \lstinline!TestCase! annotation is missing it is a normal model -- there are thus no restrictions on the use of the model, and the model shall not contain errors.
+
```

Comment:
Wouldn't it be more clear to give this as the semantics of `shouldPass = true`, and then make this the default?

Reply:
No, as the test-cases are also restricted so that they shouldn't be used in other models (even if they should pass).
That allows the test-cases to be modified without having to care about conversions, and separates out test-cases from other models.